airbrakes/state.py

```diff
@@ -59,11 +59,9 @@ class StandByState(State):
 
     __slots__ = ()
 
-    @override
```

Comment:
why?

Reply:
When looking on the docs and various stack overflow posts, I never saw people using @override with ABC. It's just not necessary/part of ABC